venues/ICLR.cc/2018/Conference/webfield/conferenceWebfield_tabs.js

```diff
@@ -95,7 +95,18 @@ function renderConferenceHeader() {
     location: "Vancouver Convention Center, Vancouver, BC, Canada",
     date: "April 30 - May 3, 2018",
     website: "http://www.iclr.cc",
-    instructions: null,  // Add any custom instructions here. Accepts HTML
+    instructions: '<p><strong>Important Information about Anonymity:</strong><br />\
+    When you post a submission to ICLR 2018, please provide your real name and email address in the submission form below.\
+    An anonymous copy of your paper will appear in the "All Submitted Papers" tab, and will be visible to the public. \
+    The <em>original</em> version of your paper will be private, and will contain your real name(s); \
+    originals can be found in your OpenReview <a href=/tasks>Tasks page</a>.\
+    You can also access the original version of your paper by clicking the "Modifiable Original" \
+    link in the discussion forum page of your paper. <br /><br /> \
```

Comment:
Instead of using 2 brs to make a new paragraph it's better if you use `<p>` tags. Just close the previous p tag and open a new one for the new paragraph.